docs/default.nix

```
@@ -209,17 +184,38 @@ let
 
                 maintainersText =
                   if maintainers == [ ] then
-                    "This module has no [dedicated maintainers](../../modules.md#maintainers)."
+                    ""
                   else
-                    ''
-                      This module is maintained by ${renderedMaintainers},
-                      pingable via: `${ghHandles}`.
-                    '';
+                    "**Maintainers**: ${renderedMaintainers} (`${ghHandles}`)";
+
+                # Render homepages as hyperlinks in readme
+                homepage = metadata.${module}.homepage or null;
+
+                renderedHomepages = joinItems (
+                  lib.mapAttrsToList (name: url: "[${name}](${url})") homepage
+                );
+
+                homepageText =
+                  if builtins.isString homepage then
+                    "**Homepage**: [${homepage}](${homepage})\n"
+                  else if builtins.isAttrs homepage then
+                    lib.throwIf (builtins.length (builtins.attrNames homepage) == 1)
+                      "stylix: ${module}: `meta.homepage.${builtins.head (builtins.attrNames homepage)}` should be simplified to `meta.homepage`"
+                      "**Homepages**: ${renderedHomepages}\n"
+                  else
+                    "";
```

Should this else-branch throw "unexpected type", and have an explicit null check separately above?
```suggestion
                  else if homepage == null then
                    ""
                  else
                    throw "unexpected type for ${module}'s `homepage`: ${builtins.typOf homepage}";
```

(If doing this, I'd probably put the null check as the first if-branch, though?)